lact-daemon/build.rs

```diff
@@ -22,6 +22,9 @@ fn gen_intel_bindings() {
         .header("include/intel.h")
         .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()))
         .dynamic_library_name("IntelDrm")
+        // Don't generate bindings for functions and types related to file access
+        .allowlist_item("(DRM|drm).*")
+        .blocklist_item(".*(FILE|file).*")
```

Comment:
The generated functions don't get called, what's the harm in having them? It might be confusing in the future if some function that's present in the header is missing in the bindings